File: app/src/main/java/com/rdev/tryp/model/favourite_driver/FavouriteDriver.java
package com.rdev.tryp.model.favourite_driver;

import com.google.gson.annotations.SerializedName;

public class FavouriteDriver {

    @SerializedName("data")
    private Data data;

    public void setData(Data data) {
        this.data = data;
    }

    public Data getData() {
        return data;
    }

    @Override
    public String toString() {
        return
                "FavouriteDriver{" +
                        "data = '" + data + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/rdev/tryp/model/favourite_driver/FavouriteDriver.kt
package com.rdev.tryp.model.favourite_driver

import com.google.gson.annotations.SerializedName


class FavouriteDriver {

    @SerializedName("data")
    var data: Data? = null

    override fun toString(): String {
        return "FavouriteDriver{" +
                "data = '" + data + '\''.toString() +
                "}"
    }

}